spec/whois.md

```diff
@@ -0,0 +1,86 @@
+## The `whois` Folder
+
+The `did:webs` method defines that a controller MAY place a folder called
+`whois` within the root folder of the `did:webs` web location and place into
+that [signed files](#signed-files) that are [verifiable credentials] (VCs) or
+[verifiable presentations] (VPs) in which the `did:webs` DID is the subject of the
+verifiable credential. This convention enables those that have the DID for the
+controller to retrieve and verify any VCs or VPs the DID controller has
+decided to publish about itself. The intention is that anyone interested in a
+particular `did:webs` DID can see if there is a `whois` web folder, and if so,
+see if it contains any useful (to the resolver) VCs or VPs that might help
+in learning more about the controller of the DID (the subject of the VCs or VPs).
+
+### `whois` Use Case
+
+The following is a use case for this capability. Consider an example of the
+`did:webs` controller being an educational institution that issues "degree"
+verifiable credentials to its graduate students. A graduate from the school
+submits as part of their job application to a company a [verifiable
+presentation] derived from the verifiable credential they received from the
+school. The company receiving the presentation can verify the cryptography of
+the presentation, but can they trust the school that issued the verifiable
+credential? If the school issued the verifiable credential using its `did:webs`
+DID, the company can resolve the DID (included in the verifiable presentation
+from the student), and verify the [[ref: KEL]]. Then it can look in the DID's
+`whois` folder where it might find a couple of useful VCs or VPs with the
+`did:webs` DID as the subject. For example:
+
+- A verifiable presentation derived from a credential issued by [GLEIF]
```

Comment:
GLEIF is very careful NOT to be the issuer of LEIs or of vLEIs. This is a deliberate choice on their part -- so they would not like this example. A more realistic example might be: "A verifiable presentation derived from a credential containing the [LEI] of the school." (just omit the "issued by GLEIF" part)

Reply:
Let's discuss this on a call. I was careful to word it in a way that I understand to be accurate.  But easily changed it not.